src/System.ServiceProcess.ServiceController/src/Configurations.props

```diff
@@ -3,6 +3,7 @@
   <PropertyGroup>
     <PackageConfigurations>
       netstandard;
+      netstandard-Windows_NT;
       netcoreapp2.0-Windows_NT;
```

Comment:
again netcoreapp2.0 config may be redundant.

Reply:
same as the other, we need it because we are harvesting `runtimes/win/lib/netstandard1.5`, so it is required in order to not have a ref impl mismatch

Reply:
I understand you need netstandard-windows_nt but I don’t think you need Netcoreapp2.0-windows_nt if that is effectively the same config as netstandard-windows_nt

Reply:
oh, I see what you mean, sure I can try get rid of that. I thought you meant the one I added was redundant

Reply:
Same on here. netcoreapp2.0 win implementation does actually contain some code on it, while the netstandard2.0 win is only a PNS assembly.